docs/snippets/all/tutorials/send_columns.py

```diff
@@ -1,17 +1,17 @@
 #!/usr/bin/env python3
-"""Very minimal test of using the temporal batch APIs.."""
+"""Very minimal test of using the send column APIs."""
 
 from __future__ import annotations
 
 import numpy as np
 import rerun as rr
 
-rr.init("rerun_example_temporal_batch", spawn=True)
+rr.init("rerun_example_send_columns", spawn=True)
 
 times = np.arange(0, 64)
 scalars = np.sin(times / 10.0)
 
-rr.log_temporal_batch(
+rr.send_columns(
     "scalars",
     times=[rr.TimeSequenceBatch("step", times)],
```

Comment:
@jleibs: did you want to change this name as well?

Reply:
Probably for consistency.

Reply:
hmmm are these then `TimeSequenceColumn`?